python/README.md

```diff
@@ -25,6 +25,8 @@ Why you should use Lance
 pip install pylance
 ```
 
+Make sure you have a recent version of pandas (1.5+), pyarrow (10.0+), and DuckDB (7.0+)
```

Comment:
Should we add this to pylance's pyproject.toml as dependency restriction?

Reply:
done